ports/nlohmann-json/vcpkg.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "nlohmann-json",
+  "version-string": "3.10.0",
```

```suggestion
  "version-semver": "3.10.0",
```
This should be served by [version-semver](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-semver).

---

yeah, wasn't sure because i haven't found a clear statement at the library. But just searched the issues and the maintainer confirmed semver https://github.com/nlohmann/json/issues/721#issuecomment-326494840.